react/build.sh

```diff
@@ -7,5 +7,5 @@ set -e # exit immediately if any command exits with a non-zero status
 rm -rf build
 # npm ci does not update minor versions ->
 # (1) less chance of breaking (2) less noise in PR from package-lock.json
-npm ci
-npm run build # defined in 'scripts' in package.json
+yarn
```

Comment:
Please add `--frozen-lockfile` 🙏 (see comment in code above) 
we actually had cases where updating minor versions broke things

Reply:
Resolved in https://github.com/sentry-demos/empower/pull/413/commits/675eee36eb5f04bac4dee0fd8c8a6a3ac7867584